### ISSUE TITLE
PC Návod

### ISSUE THREAD

**Issue by Juraj326** [TIS2024-FMFI/gastro#51]
3.3.18. Pribudne návod, ktorý používateľovi vysvetlí ako správne vyplniť
formulár

Relevantne subory:
* **resources/views/business-trips/create.blade.php** do bloku `<x-content-box title="Nová pracovná cesta">` pridat tlacidlo navod, ktore zobrazi navod/presmeruje na stranku s navodom
* **resources/views/business-trips/edit.blade.php** ak to ma byt viditelne aj v edit mode tak to iste sem
* **resources/views/business-trips** bude potrebne pridat file na generovanie HTML pre stranku s navodom, tento adresar je na to asi najvhodnejsi
* **routes/web.php** `Route::controller(BusinessTripController::class)` bude potrebne pridat routovanie na stranku s navodom

**Comment by Juraj326** [comment]
- [ ] Vyzaduje splnenie #60

**Comment by LeaMichalikova** [comment]
Predstavujem si, že sa to bude robiť podobne, ako je ukázané v dokumentácii z projektu z predošlých rokov - tlačidlo návrh zobrazí tektové bublinky (obrázky) s popsiným textom hneď vedľa komponentov, ktorých funkcionalitu vysvetľujú. Po opätovnom stlačení tlačidla textové bublinky zmiznú.

**Comment by Juraj326** [comment]
Ja to navrhujem spraviť cez ten hint element, na ktorý keď sa hoverne, tak zobrazí také to textové pole, v ktorom bude napísané, čo je potrebné doplniť do príslušného poľa formulára. Neviem ale ako presne sa ten element volá.

**Comment by LeaMichalikova** [comment]
Tento nápad sa mi páči viac, pozriem sa na to. Ďakujem za návrh.

**Comment by LeaMichalikova** [comment]
Vo formulári teda pribudla červená ikona s otáznikom, na ktorú keď sa klikne, zobrazí sa text s návodom. Nepodarilo sa mi spojazdniť funkcionality pre hover, tak je to spravené cez klikanie.

**Comment by LeaMichalikova** [comment]
Predpokladám, že po odprezentovaní sekretárke sa v návode ešte budú robiť zmeny. Zrejme tlačidlo pribudne aj na iné miesta a možno sa zmení text, ktorý som tam teraz napísala.

**Comment by LeaMichalikova** [comment]
Beriem späť, zabudla som políčko s návodom schovať keď sa jedná o tuzemskú cestu. Napravím to hneď ako budem môcť.

**Comment by Cerulikova** [comment]
Je toto teda pripravené na kontrolu, alebo tam chceš ešte niečo robiť?

**Comment by LeaMichalikova** [comment]
Zabudla som dať vedieť, pardon. Áno, je to pripravené na kontrolu.

**Comment by Cerulikova** [comment]
Ty si urobila iba ten jeden otáznik tam?

Akože vyzerá to dobre a páči sa mi to, ale je jasné, že to treba porobiť skoro ku všetkému v tom formulári na tvorbu a editovanie cesty, však? Neviem teda, či to mám mergovať, keď je to v štádiu "ukážka ako to bude všade spravené".

**Comment by LeaMichalikova** [comment]
V poznámkach zo stretnutia so zadávateľkou, kde nám zadala spraviť takýto návrh, sa hovorilo len o tomto jednom poli. Ostatné polia buď už majú vlastný dodatočný popis, alebo mi prišli príliš priamočiaré na to, aby bol pri nich návod (napr polia Meno, Priezvisko, ...). Ako som spomínala vyššie (https://github.com/TIS2024-FMFI/gastro/issues/51#issuecomment-2560265460), očakávam, že zmeny na základe pripomienok ešte nastanú zmeny, no momentálne je samotná požiadavka splnená.

**Comment by Cerulikova** [comment]
No, dobre. Tak ja to teda mergnem a potom prípadne vytvoríme novú vetvu, v ktorej podopĺňame prípadné ďalšie nápovedy.

**Comment by LeaMichalikova** [comment]
Ako bolo predpokladané, návod sa zišlo doplniť ešte na jedno miesto k číslu IBAN.

**Comment by LeaMichalikova** [comment]
Je to pripravené na merge, prosím teda okontrolovať či formulácia textu vyhovuje. [PR](https://github.com/TIS2023-FMFI/pracovne-cesty/pull/188#issue-2800233518)

**Comment by Cerulikova** [comment]
mergnute
